example_config.yaml

```diff
@@ -0,0 +1,78 @@
+# Top-level parameters
+
+exp-name: E-MTAB-1234
+droplet: False
+
+# Matrices, slots correspond to keys in .layers, unless .X or .raw.X, which
+# which case those are interpreted as explicit top-level slots in the anndata.
+
+matrices:
+  - slot: raw.X
+    measure: counts
+    cell_filtered: True
+    gene_filtered: False
+    normalised: False
+    log_transformed: False
+
+  - slot: filtered
+    measure: counts
+    cell_filtered: True
+    gene_filtered: True
+    normalised: False
+    log_transformed: False
+    
+    parameters:
+        min_count: 3
+  
+  - slot: normalised
+    measure: counts
+    cell_filtered: True
+    gene_filtered: True
+    normalised: True
+    log_transformed: True
+    
+
+# Cell groups, slots correspond to columns in .obs. Two categories:
+#     - curated: present before analysis. These will be output for curation.
+#     - analysis: derived from analysis: e.g. clusters.  
+# 
+# Cell groupings marked with 'markers: True' will have those markers written to
+# the bundle
+
+cell_groups:
+  - slot: organism_part
+    kind: curation
+    markers: False
+
+  - slot: louvain_resolution_0.7
+    kind: analysis
+    markers: True
+    parameters:
+      alorithm: louvain
+      resolution: 0.7
+      
+  - slot: louvain_resolution_1.0
+    kind: analysis
+    markers: True
+    parameters:
+      alorithm: louvain
+      resolution: 1.0
+
```

Comment:
I wonder if we need to specify any relevant metadata elements? Maybe the name of certain columns like gene identifiers, gene symbol, source of gene identifier annotation (so that we can translate into ensemble gene ids), etc?

Reply:
Yes, but this would be a top-level thing (right?) rather than associated with cell groupings as you've placed this comment.